File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_cardfeein.cs
// EXPECTED: 44.44
// Reference emit for CARDFEEIN — CardDemo fee→interest multi-COMPUTE chain.
using System;
public static class CardfeeinRef {
  public static void Main() {
    double bal = 1000.00;
    double feeRate = 0.0290;
    double intRate = 0.0150;
    double fee = Math.Round(bal * feeRate, 2);
    double interest = Math.Round((bal + fee) * intRate, 2);
    double total = Math.Round(fee + interest, 2);
    Console.WriteLine(total.ToString("0.00"));
  }
}

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_cardfeein.py
"""Reference emit for CARDFEEIN — CardDemo fee→interest chain → 44.44."""
# EXPECTED: 44.44
BAL = 1000.00
FEE_RATE = 0.0290
INT_RATE = 0.0150
FEE = round(BAL * FEE_RATE + 1e-12, 2)
INTEREST = round((BAL + FEE) * INT_RATE + 1e-12, 2)
TOTAL = round(FEE + INTEREST + 1e-12, 2)
print(f"{TOTAL:.2f}")
